src/hooks/use-all-media.js

```diff
@@ -0,0 +1,46 @@
+import { useEffect, useState } from 'react';
+import fetchTwitchVideos from '../utils/fetch-twitch-videos';
+import fetchYoutubeData from '../utils/fetch-youtube-data';
+import fetchLybsinPodcasts from '../utils/fetch-lybsin-podcasts';
+import { TWITCH_HEADERS, TWITCH_VIDEO_URL } from '../constants';
+
+const useAllMedia = () => {
+    const [media, setMedia] = useState(null);
+    const [error, setError] = useState(null);
+    const [isLoading, setIsLoading] = useState(false);
+    const YT_PLAYLIST = 'PLN3n1USn4xlmyw3ebYuZmGp60mcENitdM';
+
+    useEffect(() => {
+        const getMedia = async () => {
+            setIsLoading(true);
+            try {
+                const youtubeVideos = fetchYoutubeData(YT_PLAYLIST);
+                const twitchVideos = fetchTwitchVideos(
+                    TWITCH_HEADERS,
+                    TWITCH_VIDEO_URL
```

Comment:
Do we also want to up or remove the `videoLimit` for twitch videos? Right now we get one, but we likely want all. We might need to play with `video_URL` (let's make this `videoURL`) in that file to do so.

Reply:
I got rid of the optional `first` query parameter from the url. That should retrieve all of the videos for us.